lib/database/adapters/DataPassAdapter.js

```diff
@@ -35,6 +35,7 @@ class DataPassAdapter {
             name,
             versions,
             skimmingStage,
+            frozen,
```

Comment:
Normally for boolean values I would recommend using the descriptive convention, in this case`isFrozen`.
What do you think?

Reply:
I agree, done